---
title: Publications
#subtitle: INRAE Micalis institute Paris Saclay
layout: page
#callouts: home_callouts
hero_image: /img/FME_Banner_website_V3.png
hero_darken: fasle
show_sidebar: false
sitemap:
  exclude: 'yes'
---

*Last extraction 2023-07-01 00:46:49.036624*

<span class="csl-left-margin">1. </span><span
class="csl-right-inline">Champomier-Vergès, M.-C. *et al.* [Le potentiel
insoupçonné des aliments fermentés](https://hal.inrae.fr/hal-03958354).
*Ressources, la revue INRAE* 44 (2023).</span>

<span class="csl-left-margin">2. </span><span
class="csl-right-inline">Ladeira, R., Tap, J. & Derrien, M. [Exploring
Bifidobacterium species community and functional variations with human
gut microbiome structure and health beyond
infancy](https://doi.org/10.20517/mrr.2023.01). *Microbiome Research
Reports* **2**, 9 (2023).</span>

<span class="csl-left-margin">3. </span><span
class="csl-right-inline">Ammoun, I. *et al.* [Lebanese fermented goat
milk products: From tradition to
meta-omics](https://doi.org/10.1016/j.foodres.2023.112762). *Food
Research International* **168**, 112762 (2023).</span>

<span class="csl-left-margin">4. </span><span
class="csl-right-inline">Rodriguez, C. I. *et al.* [Curated and
harmonized gut microbiome 16S <span class="nocase">rRNA</span> amplicon
data from dietary fiber intervention studies in
humans](https://doi.org/10.1038/s41597-023-02254-4). *Scientific Data*
**10**, 346 (2023).</span>

<span class="csl-left-margin">5. </span><span
class="csl-right-inline">Tap, J. *et al.* [Global branches and local
states of the human gut microbiome define associations with
environmental and intrinsic
factors](https://doi.org/10.1038/s41467-023-38558-7). *Nature
Communications* **14**, 3310 (2023).</span>

<span class="csl-left-margin">6. </span><span
class="csl-right-inline">Tap, J. *et al.* [Global branches and local
states of the human gut microbiome define associations with
environmental and intrinsic
factors](https://doi.org/10.1038/s41467-023-38558-7). *Nature
Communications* **14**, 3310 (2023).</span>

<span class="csl-left-margin">7. </span><span
class="csl-right-inline">Kothe, C. I., Mohellibi, N. & Renault, P.
[Revealing the microbial heritage of traditional Brazilian cheeses
through metagenomics](https://doi.org/10.1016/j.foodres.2022.111265).
*Food Research International (Ottawa, Ont.)* **157**, 111265
(2022).</span>

<span class="csl-left-margin">8. </span><span
class="csl-right-inline">Claus, P. *et al.* [Discrimination of spoiled
beef and salmon stored under different atmospheres by an optoelectronic
nose. Comparison with GC-MS
measurements](https://doi.org/10.1016/j.fufo.2021.100106). *Future
Foods* **5**, 100106 (2022).</span>

<span class="csl-left-margin">9. </span><span
class="csl-right-inline">Manthou, E., Coeuret, G., Chaillou, S. &
Nychas, G.-J. E. [Metagenetic characterization of bacterial communities
associated with ready-to-eat leafy vegetables and study of temperature
effect on their composition during
storage](https://doi.org/10.1016/j.foodres.2022.111563). *Food Research
International (Ottawa, Ont.)* **158**, 111563 (2022).</span>

<span class="csl-left-margin">10. </span><span
class="csl-right-inline">Borges, F. *et al.* [Contribution of omics to
biopreservation: Toward food microbiome
engineering](https://doi.org/10.3389/fmicb.2022.951182). *Frontiers in
Microbiology* **13**, 951182 (2022).</span>

<span class="csl-left-margin">11. </span><span
class="csl-right-inline">Rul, F. *et al.* [Underlying evidence for the
health benefits of fermented foods in
humans](https://doi.org/10.1039/D1FO03989J). *Food & Function* **13**,
4804–4824 (2022).</span>

<span class="csl-left-margin">12. </span><span
class="csl-right-inline">Lange, L. *et al.* [Microbiome ethics, guiding
principles for microbiome research, use and knowledge
management](https://doi.org/10.1186/s40793-022-00444-y). *Environmental
Microbiome* **17**, 50 (2022).</span>

<span class="csl-left-margin">13. </span><span
class="csl-right-inline">Cernava, T. *et al.* [Metadata
harmonization–Standards are the key for a better usage of omics data for
integrative microbiome
analysis](https://doi.org/10.1186/s40793-022-00425-1). *Environmental
Microbiome* **17**, 33 (2022).</span>

<span class="csl-left-margin">14. </span><span
class="csl-right-inline">Grassotti, T. T. *et al.* [Fecal bacterial
communities of wild black capuchin monkeys (Sapajus nigritus) from the
Atlantic Forest biome in Southern Brazil are divergent from those of
other non-human primates](https://doi.org/10.1016/j.crmicr.2021.100048).
*Current Research in Microbial Sciences* **2**, 100048 (2021).</span>

<span class="csl-left-margin">15. </span><span
class="csl-right-inline">Kothe, C. I., Bolotin, A., Kraïem, B.-F.,
Dridi, B. & Renault, P. [Unraveling the world of halophilic and
halotolerant bacteria in cheese by combining cultural, genomic and
metagenomic
approaches.](https://doi.org/10.1016/j.ijfoodmicro.2021.109312)
*International journal of food microbiology* **358**, 109312
(2021).</span>

<span class="csl-left-margin">16. </span><span
class="csl-right-inline">Luong, N.-D. M. *et al.* [Application of a
path-modelling approach for deciphering causality relationships between
microbiota, volatile organic compounds and off-odour profiles during
meat spoilage](https://doi.org/10.1016/j.ijfoodmicro.2021.109208).
*International Journal of Food Microbiology* **348**, 109208
(2021).</span>

<span class="csl-left-margin">17. </span><span
class="csl-right-inline">Manthou, E., Coeuret, G., Chaillou, S. &
Nychas, G.-J. E. [Evolution of fungal community associated with
ready-to-eat pineapple during storage under different temperature
conditions](https://doi.org/10.1016/j.fm.2021.103736). *Food
Microbiology* **97**, 103736 (2021).</span>

<span class="csl-left-margin">18. </span><span
class="csl-right-inline">Verplaetse, E. *et al.* [Heme Uptake in
Lactobacillus sakei Evidenced by a New Energy Coupling Factor (ECF)-Like
Transport System.](https://doi.org/10.1128/AEM.02847-19) *Applied and
environmental microbiology* **86**, (2020).</span>

<span class="csl-left-margin">19. </span><span
class="csl-right-inline">Zagdoun, M., Coeuret, G., N’Dione, M.,
Champomier-Vergès, M.-C. & Chaillou, S. [Large microbiota survey reveals
how the microbial ecology of cooked ham is shaped by different
processing steps.](https://doi.org/10.1016/j.fm.2020.103547) *Food
microbiology* **91**, 103547 (2020).</span>

<span class="csl-left-margin">20. </span><span
class="csl-right-inline">Poirier, S. *et al.* [Large-scale multivariate
dataset on the characterization of microbiota diversity, microbial
growth dynamics, metabolic spoilage volatilome and sensorial profiles of
two industrially produced meat products subjected to changes in lactate
concentration and packaging
atmosphere.](https://doi.org/10.1016/j.dib.2020.105453) *Data in brief*
**30**, 105453 (2020).</span>

<span class="csl-left-margin">21. </span><span
class="csl-right-inline">Terán, L. C. *et al.* [Nucleotide sequence and
analysis of <span class="nocase">pRC12</span> and <span
class="nocase">pRC18</span>, two theta-replicating plasmids harbored by
Lactobacillus curvatus CRL
705.](https://doi.org/10.1371/journal.pone.0230857) *PloS one* **15**,
e0230857 (2020).</span>

<span class="csl-left-margin">22. </span><span
class="csl-right-inline">Papadimitriou, K., Kline, K., Renault, P. &
Kok, J. [Editorial: Omics and Systems Approaches to Study the Biology
and Applications of Lactic Acid
Bacteria.](https://doi.org/10.3389/fmicb.2020.01786) *Frontiers in
microbiology* **11**, 1786 (2020).</span>

<span class="csl-left-margin">23. </span><span
class="csl-right-inline">Sanhoun, A. R. *et al.* [Traditional milk
transformation schemes in Côte d’Ivoire and their impact on the
prevalence of Streptococcus bovis complex bacteria in dairy
products.](https://doi.org/10.1371/journal.pone.0233132) *PloS one*
**15**, e0233132 (2020).</span>

<span class="csl-left-margin">24. </span><span
class="csl-right-inline">Aka, S. *et al.* [Characterization of lactic
acid bacteria isolated from a traditional Ivoirian beer process to
develop starter cultures for safe sorghum-based
beverages.](https://doi.org/10.1016/j.ijfoodmicro.2020.108547)
*International journal of food microbiology* **322**, 108547
(2020).</span>

<span class="csl-left-margin">25. </span><span
class="csl-right-inline">Kothe, C. I., Delbarre-Ladrat, C., Renault, P.
& Passerini, D. [Draft-genome sequence data and phylogenomic comparison
of two marine-sourced bacterial strains Pseudoalteromonas sp. MIP2626
and Psychrobacter sp. BI730.](https://doi.org/10.1016/j.dib.2020.105898)
*Data in brief* **31**, 105898 (2020).</span>

<span class="csl-left-margin">26. </span><span
class="csl-right-inline">Dugat-Bony, E. *et al.* [Viral metagenomic
analysis of the cheese surface: A comparative study of rapid procedures
for extracting viral
particles](https://doi.org/10.1016/j.fm.2019.103278). *Food
Microbiology* **85**, 103278 (2020).</span>

<span class="csl-left-margin">27. </span><span
class="csl-right-inline">Gboko, K. D. T. *et al.* [Risk factors for the
carriage of Streptococcus infantarius subspecies infantarius isolated
from African fermented dairy
products.](https://doi.org/10.1371/journal.pone.0225452) *PloS one*
**14**, e0225452 (2019).</span>

<span class="csl-left-margin">28. </span><span
class="csl-right-inline">Terán, L. C. *et al.* [Phylogenomic Analysis of
Lactobacillus curvatus Reveals Two Lineages Distinguished by Genes for
Fermenting Plant-Derived
Carbohydrates.](https://doi.org/10.1093/gbe/evy106) *Genome biology and
evolution* **10**, 1516–1525 (2018).</span>

<span class="csl-left-margin">29. </span><span
class="csl-right-inline">Couvigny, B. *et al.* [Identification of New
Factors Modulating Adhesion Abilities of the Pioneer Commensal Bacterium
Streptococcus salivarius.](https://doi.org/10.3389/fmicb.2018.00273)
*Frontiers in microbiology* **9**, 273 (2018).</span>

<span class="csl-left-margin">30. </span><span
class="csl-right-inline">Isenring, J. *et al.* [Streptococcus
gallolyticus subsp. Gallolyticus endocarditis isolate interferes with
coagulation and activates the contact
system.](https://doi.org/10.1080/21505594.2017.1393600) *Virulence*
**9**, 248–261 (2018).</span>

<span class="csl-left-margin">31. </span><span
class="csl-right-inline">Kaindi, D. W. M. *et al.* [Colorectal
cancer-associated Streptococcus infantarius subsp. Infantarius differ
from a major dairy lineage providing evidence for pathogenic, pathobiont
and food-grade lineages.](https://doi.org/10.1038/s41598-018-27383-4)
*Scientific reports* **8**, 9181 (2018).</span>

<span class="csl-left-margin">32. </span><span
class="csl-right-inline">Kaindi, D. W. M. *et al.* [Investigating the
association between African spontaneously fermented dairy products,
faecal carriage of Streptococcus infantarius subsp. Infantarius and
colorectal adenocarcinoma in
Kenya.](https://doi.org/10.1016/j.actatropica.2017.10.018) *Acta
tropica* **178**, 10–18 (2018).</span>

<span class="csl-left-margin">33. </span><span
class="csl-right-inline">Jans, C. *et al.* [African fermented dairy
products - Overview of predominant technologically important
microorganisms focusing on African Streptococcus infantarius variants
and potential future applications for enhanced food safety and
security.](https://doi.org/10.1016/j.ijfoodmicro.2017.03.012)
*International journal of food microbiology* **250**, 27–36
(2017).</span>

<span class="csl-left-margin">34. </span><span
class="csl-right-inline">Delorme, C. *et al.* [Study of Streptococcus
thermophilus population on a world-wide and historical collection by a
new MLST scheme.](https://doi.org/10.1016/j.ijfoodmicro.2016.11.016)
*International journal of food microbiology* **242**, 70–81
(2017).</span>

<span class="csl-left-margin">35. </span><span
class="csl-right-inline">Couvigny, B. *et al.* [Three glycosylated
serine-rich repeat proteins play a pivotal role in adhesion and
colonization of the pioneer commensal bacterium, Streptococcus
salivarius.](https://doi.org/10.1111/1462-2920.13853) *Environmental
microbiology* **19**, 3579–3594 (2017).</span>

<span class="csl-left-margin">36. </span><span
class="csl-right-inline">Fougy, L. *et al.* [Reducing Salt in Raw Pork
Sausages Increases Spoilage and Correlates with Reduced Bacterial
Diversity.](https://doi.org/10.1128/AEM.00323-16) *Applied and
environmental microbiology* **82**, 3928–3939 (2016).</span>

<span class="csl-left-margin">37. </span><span
class="csl-right-inline">Papadimitriou, K., Mavrogonatou, E., Bolotin,
A., Tsakalidou, E. & Renault, P. [Whole-Genome Sequence of the Cheese
Isolate Streptococcus macedonicus
679.](https://doi.org/10.1128/genomeA.01025-16) *Genome announcements*
**4**, (2016).</span>

<span class="csl-left-margin">38. </span><span
class="csl-right-inline">Chaillou, S. *et al.* [Origin and ecological
selection of core and food-specific bacterial communities associated
with meat and seafood spoilage.](https://doi.org/10.1038/ismej.2014.202)
*The ISME journal* **9**, 1105–1118 (2015).</span>

<span class="csl-left-margin">39. </span><span
class="csl-right-inline">Delorme, C., Abraham, A.-L., Renault, P. &
Guédon, E. [Genomics of Streptococcus salivarius, a major human
commensal.](https://doi.org/10.1016/j.meegid.2014.10.001) *Infection,
genetics and evolution : journal of molecular epidemiology and
evolutionary genetics in infectious diseases* **33**, 381–392
(2015).</span>

<span class="csl-left-margin">40. </span><span
class="csl-right-inline">Couvigny, B. *et al.* [Streptococcus
thermophilus Biofilm Formation: A Remnant Trait of Ancestral Commensal
Life?](https://doi.org/10.1371/journal.pone.0128099) *PloS one* **10**,
e0128099 (2015).</span>

<span class="csl-left-margin">41. </span><span
class="csl-right-inline">Couvigny, B. *et al.* [Commensal Streptococcus
salivarius Modulates PPARγ Transcriptional Activity in Human Intestinal
Epithelial Cells.](https://doi.org/10.1371/journal.pone.0125371) *PloS
one* **10**, e0125371 (2015).</span>

<span class="csl-left-margin">42. </span><span
class="csl-right-inline">Papadimitriou, K. *et al.* [Acquisition through
horizontal gene transfer of plasmid <span class="nocase">pSMA198</span>
by Streptococcus macedonicus ACA-DC 198 points towards the dairy origin
of the species.](https://doi.org/10.1371/journal.pone.0116337) *PloS
one* **10**, e0116337 (2015).</span>

<span class="csl-left-margin">43. </span><span
class="csl-right-inline">Chaillou, S. *et al.* [Quantification and
efficiency of Lactobacillus sakei strain mixtures used as protective
cultures in ground beef.](https://doi.org/10.1016/j.meatsci.2013.08.009)
*Meat science* **97**, 332–338 (2014).</span>

<span class="csl-left-margin">44. </span><span
class="csl-right-inline">Nielsen, H. B. *et al.* [Identification and
assembly of genomes and genetic elements in complex metagenomic samples
without using reference genomes.](https://doi.org/10.1038/nbt.2939)
*Nature biotechnology* **32**, 822–828 (2014).</span>

<span class="csl-left-margin">45. </span><span
class="csl-right-inline">Cheeseman, K. *et al.* [Multiple recent
horizontal transfers of a large genomic region in cheese making
fungi.](https://doi.org/10.1038/ncomms3876) *Nature communications*
**5**, 2876 (2014).</span>

<span class="csl-left-margin">46. </span><span
class="csl-right-inline">Kaci, G. *et al.* [Anti-inflammatory properties
of Streptococcus salivarius, a commensal bacterium of the oral cavity
and digestive tract.](https://doi.org/10.1128/AEM.03133-13) *Applied and
environmental microbiology* **80**, 928–934 (2014).</span>

<span class="csl-left-margin">47. </span><span
class="csl-right-inline">Ceuppens, S. *et al.* [Molecular Methods in
Food Safety Microbiology: Interpretation and Implications of Nucleic
Acid Detection.](https://doi.org/10.1111/1541-4337.12072) *Comprehensive
reviews in food science and food safety* **13**, 551–577 (2014).</span>

<span class="csl-left-margin">48. </span><span
class="csl-right-inline">Almeida, M. *et al.* [Construction of a dairy
microbial genome catalog opens new perspectives for the metagenomic
analysis of dairy fermented
products.](https://doi.org/10.1186/1471-2164-15-1101) *BMC genomics*
**15**, 1101 (2014).</span>

<span class="csl-left-margin">49. </span><span
class="csl-right-inline">Papadimitriou, K. *et al.* [Comparative
genomics of the dairy isolate Streptococcus macedonicus ACA-DC 198
against related members of the Streptococcus bovis/Streptococcus equinus
complex.](https://doi.org/10.1186/1471-2164-15-272) *BMC genomics*
**15**, 272 (2014).</span>

<span class="csl-left-margin">50. </span><span
class="csl-right-inline">Chaillou, S., Lucquin, I., Najjari, A.,
Zagorec, M. & Champomier-Vergès, M.-C. [Population genetics of
Lactobacillus sakei reveals three lineages with distinct evolutionary
histories.](https://doi.org/10.1371/journal.pone.0073253) *PloS one*
**8**, e73253 (2013).</span>

<span class="csl-left-margin">51. </span><span
class="csl-right-inline">Cotillard, A. *et al.* [Dietary intervention
impact on gut microbial gene
richness.](https://doi.org/10.1038/nature12480) *Nature* **500**,
585–588 (2013).</span>

<span class="csl-left-margin">52. </span><span
class="csl-right-inline">Le Chatelier, E. *et al.* [Richness of human
gut microbiome correlates with metabolic
markers.](https://doi.org/10.1038/nature12506) *Nature* **500**, 541–546
(2013).</span>

<span class="csl-left-margin">53. </span><span
class="csl-right-inline">Morrison, D. A., Guédon, E. & Renault, P.
[Competence for natural genetic transformation in the Streptococcus
bovis group streptococci S. Infantarius and S.
macedonicus.](https://doi.org/10.1128/JB.00230-13) *Journal of
bacteriology* **195**, 2612–2620 (2013).</span>

<span class="csl-left-margin">54. </span><span
class="csl-right-inline">Guilbaud, M., Zagorec, M., Chaillou, S. &
Champomier-Vergès, M.-C. [Intraspecies diversity of Lactobacillus sakei
response to oxidative stress and variability of strain performance in
mixed strains challenges.](https://doi.org/10.1016/j.fm.2011.07.011)
*Food microbiology* **29**, 197–204 (2012).</span>

<span class="csl-left-margin">55. </span><span
class="csl-right-inline">Lucquin, I., Zagorec, M., Champomier-Vergès, M.
& Chaillou, S. [Fingerprint of lactic acid bacteria population in beef
carpaccio is influenced by storage process and seasonal
changes.](https://doi.org/10.1016/j.fm.2011.08.001) *Food microbiology*
**29**, 187–196 (2012).</span>

<span class="csl-left-margin">56. </span><span
class="csl-right-inline">Meslier, V., Loux, V. & Renault, P. [Genome
sequence of Leuconostoc pseudomesenteroides strain 4882, isolated from a
dairy starter culture.](https://doi.org/10.1128/JB.01696-12) *Journal of
bacteriology* **194**, 6637 (2012).</span>

<span class="csl-left-margin">57. </span><span
class="csl-right-inline">Chaillou, S. *et al.* [Intraspecies genomic
diversity and natural population structure of the meat-borne lactic acid
bacterium Lactobacillus sakei.](https://doi.org/10.1128/AEM.01721-08)
*Applied and environmental microbiology* **75**, 970–980 (2009).</span>

<span class="csl-left-margin">58. </span><span
class="csl-right-inline">Chaillou, S. *et al.* [The complete genome
sequence of the meat-borne lactic acid bacterium Lactobacillus sakei
23K.](https://doi.org/10.1038/nbt1160) *Nature biotechnology* **23**,
1527–1533 (2005).</span>

<span class="csl-left-margin">59. </span><span
class="csl-right-inline">Marceau, A., Zagorec, M., Chaillou, S., Méra,
T. & Champomier-Vergès, M.-C. [Evidence for involvement of at least six
proteins in adaptation of Lactobacillus sakei to cold temperatures and
addition of NaCl.](https://doi.org/10.1128/AEM.70.12.7260-7268.2004)
*Applied and environmental microbiology* **70**, 7260–7268
(2004).</span>

<span class="csl-left-margin">60. </span><span
class="csl-right-inline">Dudez, A.-M. *et al.* [Physical and genetic map
of the Lactobacillus sakei 23K
chromosome.](https://doi.org/10.1099/00221287-148-2-421) *Microbiology
(Reading, England)* **148**, 421–431 (2002).</span>

<span class="csl-left-margin">61. </span><span
class="csl-right-inline">Champomier-Vergès, M. C., Chaillou, S., Cornet,
M. & Zagorec, M. [Lactobacillus sakei: Recent developments and future
prospects.](https://doi.org/10.1016/s0923-2508(01)01267-0) *Research in
microbiology* **152**, 839–848 (2001).</span>
